Listener panic

Today the listener stopped working. Just using the main.go (with authentication disabled), the latest commit causes a panic:
```
> go run main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x58572f]

goroutine 1 [running]:
log/slog.(*Logger).clone(...)
	/usr/lib/go/src/log/slog/logger.go:116
log/slog.(*Logger).With(0x0, {0x48a3c253b90, 0x2, 0x2})
	/usr/lib/go/src/log/slog/logger.go:130 +0x4f
github.com/sandertv/gophertunnel/minecraft.RakNet.Listen({0x977400?}, {0x48a3c21e670, 0xd})
	/home/user/gophertunnel/minecraft/raknet.go:27 +0x65
github.com/sandertv/gophertunnel/minecraft.ListenConfig.ListenNetwork({0x0, 0x0, 0x0, 0x0, 0x0, 0x0, {0x96f6e0, 0x48a3c2a44e0}, {0x0, 0x0, ...}, ...}, ...)
	/home/user/gophertunnel/minecraft/listener.go:196 +0x442
github.com/sandertv/gophertunnel/minecraft.ListenConfig.Listen({0x0, 0x0, 0x0, 0x0, 0x0, 0x0, {0x96f6e0, 0x48a3c2a44e0}, {0x0, 0x0, ...}, ...}, ...)
	/home/user/gophertunnel/minecraft/listener.go:146 +0x1b2
main.main()
	/home/user/gophertunnel/main.go:32 +0xdd
exit status 2
```

```
> diff main.go main.go.orig
11,12c11
<
< 	// "github.com/sandertv/gophertunnel/minecraft/auth"
---
> 	"github.com/sandertv/gophertunnel/minecraft/auth"
19,24c18,22
< 	// token, err := auth.RequestLiveToken()
< 	// if err != nil {
< 	// 	panic(err)
< 	// }
< 	// src := auth.RefreshTokenSource(token)
< 	var src oauth2.TokenSource
---
> 	token, err := auth.RequestLiveToken()
> 	if err != nil {
> 		panic(err)
> 	}
> 	src := auth.RefreshTokenSource(token)
37d34
< 	log.Println("listening...")
50c47
< 		// TokenSource: src,
---
> 		TokenSource: src,
```

--- a/minecraft/dial.go
+++ b/minecraft/dial.go
@@ -317,6 +317,10 @@ func (d Dialer) DialContextNetwork(ctx context.Context, network Network, address
 // typically "raknet". A Conn is returned which may be used to receive packets from and send packets to.
 // If a connection is not established before the context passed is cancelled, DialContext returns an error.
 func (d Dialer) DialContext(ctx context.Context, network, address string) (conn *Conn, err error) {
+	if d.ErrorLog == nil {
+		d.ErrorLog = slog.New(internal.DiscardHandler{})
+	}
+	d.ErrorLog = d.ErrorLog.With("src", "dialer")
 	n, ok := networkByID(network, d.ErrorLog)
 	if !ok {
 		return nil, &net.OpError{Op: "dial", Net: "minecraft", Err: fmt.Errorf("dial: no network under id %v", network)}

--- a/minecraft/listener.go
+++ b/minecraft/listener.go
@@ -139,6 +139,9 @@ type Listener struct {
 // If the host in the address parameter is empty or a literal unspecified IP address, Listen listens on all
 // available unicast and anycast IP addresses of the local system.
 func (cfg ListenConfig) Listen(network string, address string) (*Listener, error) {
+	if cfg.ErrorLog == nil {
+		cfg.ErrorLog = slog.New(internal.DiscardHandler{})
+	}
 	n, ok := networkByID(network, cfg.ErrorLog)
 	if !ok {
 		return nil, fmt.Errorf("listen: no network under id %v", network)

--- a/minecraft/listener.go
+++ b/minecraft/listener.go
@@ -142,6 +142,7 @@ func (cfg ListenConfig) Listen(network string, address string) (*Listener, error
 	if cfg.ErrorLog == nil {
 		cfg.ErrorLog = slog.New(internal.DiscardHandler{})
 	}
+	cfg.ErrorLog = cfg.ErrorLog.With("src", "listener")
 	n, ok := networkByID(network, cfg.ErrorLog)
 	if !ok {
 		return nil, fmt.Errorf("listen: no network under id %v", network)

